convert.sh

```
@@ -0,0 +1,7 @@
+for f in *.md; do
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)